modules/virt-configuring-disk-sharing-lun.adoc

```diff
@@ -25,6 +26,7 @@ For a LUN disk with an iSCSi connection and a persistent reservation, as require
 If the VMs that are sharing disks are running on the same node, `ReadWriteOnce` (RWO) volume access mode is sufficient.
 * The storage provider must support a Container Storage Interface (CSI) driver that uses the SCSI protocol.
 * If you are a cluster administrator and intend to configure disk sharing by using LUN, you must enable the cluster's feature gate on the `HyperConverged` custom resource (CR).
+* Any disk that you want to share must be in block mode.
```

Comment:
Hope this sounds technically correct

```suggestion
* The shared disks must be in block mode.
```

Reply:
That could imply that the disk is already shared, but setting up the sharing is the point of the procedure, so that wording might be confusing. I'll try to come up with a good compromise.